docs/remote/devcontainer-cli.md

```diff
@@ -80,6 +99,20 @@ The `devcontainer build` command allows you to quickly build dev container image
 
 As with the `open` command, `build` accepts a path to the folder containing a `.devcontainer` folder or `.devcontainer.json` file. If omitted, the current working folder is used. For example, `devcontainer build` will build the dev container image for the current folder and `devcontainer build /source/my-folder` will build the container image for the `/source/my-folder` folder.
 
+You can use `devcontainer build -h` to see the supported `build` options.
+```bash
+Options:
+
+  -h, --help               Show help [boolean]
+      --disable-telemetry  Disable telemetry  [boolean] [default: false]
+      --verbose            Run build with increased log level  [boolean] [default: false]
+      --no-cache           Disable image cache for the dev container build [boolean] [default: false]
+      --image-name         Specify the image name  [string]
+      -- platform          Comma delimited string with multiple architectures [string]
```

Comment:
Nit: there is a space in the --platform option. The output of `devcontainer build --help` is:

```sh
Options:
  -h, --help               Show help  [boolean]
      --disable-telemetry  Disable telemetry  [boolean] [default: false]
      --verbose            Run build with increased log level  [boolean] [default: false]
      --no-cache           disable image cache for the dev container build  [boolean] [default: false]
      --image-name         specify the image name  [string]
      --platform           build platforms and push to container registry  [string]
      --no-push            do not push to container registry  [boolean]
```